flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/batch/sql/SelectITCase.scala

```diff
@@ -146,4 +148,21 @@ class SelectITCase(
     tEnv.sql(sqlQuery)
   }
 
+  @Test
+  def testConstantReduce(): Unit = {
```

Comment:
👍  It's a  good idea. I will try it later. And the CI throws `cannot translate call AS...` error, I will figure it out today.